### ISSUE TITLE
variable ‘wakeup_in’ set but not used

### ISSUE THREAD

**Issue by Sarisan** [tdlib/td#2532]
I get this compiling telegram-bot-api:
```
/home/sarisan/Documents/git/telegram-bot-api/td/td/telegram/net/Session.cpp: In member function ‘virtual void td::Session::loop()’:
/home/sarisan/Documents/git/telegram-bot-api/td/td/telegram/net/Session.cpp:1582:10: warning: variable ‘wakeup_in’ set but not used [-Wunused-but-set-variable]
 1582 |   double wakeup_in = 0;
      |          ^~~~~~~~~
```

**Distribution:** Debian 12
**Compiler:** gcc (Debian 12.2.0-14) 12.2.0
**td:** 66234ae2537a99ec0eaf7b0857245a6e5c2d2bc9

**Comment by levlam** [comment]
This is just a warning.

**Comment by Sarisan** [comment]
> This is just a warning.

So can you suppress it like you always do?

**Comment by levlam** [comment]
It was fixed 2 month ago in a05453a2c3dccee96ebc5f35d4038941dc643132, but this is not a reason to update TDLib in Telegram Bot API. You can do this locally after repository clonning.
